src/hotspot/os/linux/cgroupSubsystem_linux.cpp

```diff
@@ -661,7 +661,7 @@ bool CgroupController::read_numerical_key_value(const char* filename, const char
   for (; line != nullptr; line = fgets(buf, buf_len, fp)) {
     char after_key = line[key_len];
     if (strncmp(line, key, key_len) == 0
-          && isspace(after_key) != 0
+          && os::is_space(after_key) != 0
```

Comment:
I think this is excessive. The value is a char so cannot be a problem. The only calls to isspace that need checking are those that actually pass an int, and even then there could be adequate safeguards in place.